Mathlib/Analysis/Normed/Module/Alternating/Basic.lean

```diff
@@ -512,6 +517,44 @@ lemma fderivCompContinuousLinearMap_apply (f : F [⋀^ι]→L[𝕜] G) (g dg : E
       ∑ i, f fun j ↦ Function.update (fun _ ↦ g) i dg j (v j) := by
   simp [fderivCompContinuousLinearMap]
 
+@[nontriviality]
+lemma fderivCompContinuousLinearMap_of_isEmpty [IsEmpty ι] :
+    fderivCompContinuousLinearMap (ι := ι) (𝕜 := 𝕜) (E := E) (F := F) (G := G) = 0 := by
+  ext; simp
+
+variable (G) in
+/-- `fderivCompContinuousLinearMap` as a continuous linear map. -/
+def fderivCompContinuousLinearMapCLM (g : E →L[𝕜] F) :
+    (F [⋀^ι]→L[𝕜] G) →L[𝕜] (E →L[𝕜] F) →L[𝕜] (E [⋀^ι]→L[𝕜] G) :=
+  LinearMap.mkContinuous
+    { toFun := (fderivCompContinuousLinearMap · g)
+      map_add' f₁ f₂ := by ext; simp [Finset.sum_add_distrib]
+      map_smul' c f := by ext; simp [Finset.smul_sum] }
+    (Fintype.card ι * ‖g‖ ^ (Fintype.card ι - 1))
+    fun f ↦ by
+      refine ContinuousLinearMap.opNorm_le_bound _ (by positivity) fun dg ↦ ?_
+      refine opNorm_le_bound _ (by positivity) fun v ↦ ?_
+      simp? [mul_assoc] says
```

Comment:
My understanding is that we try not to introduce new uses of `simp? says` --- and this one certainly doesn't feel absolutely. What do you think?

Reply:
I think I've answered all other comments. I'm going to merge it when the CI is happy.